w2/row.py

```diff
@@ -2,6 +2,5 @@ class ROW:
     def __init__(self, t):
         self.cells = t
 
-    @classmethod
     def new(cls, t):
         return cls(cells=t)
```

Comment:
I don't think we need to have a new() function. We can just use default initializer to create a new class instance.

Reply:
so the init is sufficient is what you are saying right?

Reply:
Yes, since we discuessed in discord that we're going to use NUM() instead of NUM.new().